poetry/publishing/uploader.py

```diff
@@ -217,8 +220,8 @@ def _upload(
             ):
                 try:
                     self._register(session, url)
-                except HTTPError as e:
-                    raise UploadError(e)
+                except HTTPError as e2:
```

Comment:
You're going to get conflicts here most likely as this was just refactored (and it fixed the poor structure here).